tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "@vercel/style-guide/typescript",
+  "compilerOptions": {
+    "target": "esnext",
+    "module": "esnext",
+    "moduleResolution": "node",
+    "noEmit": true,
```

Comment:
I think we still need to emit JS

Reply:
I'm using `tsc` only to check types, `tsup` does the actual compilation of ts => js, so there is no need for `tsc` to emit js code.